src/components/ChartMode.tsx

```diff
@@ -64,6 +65,13 @@ const ChartMode = ({ onChartType, onGranularity, portalNodeId }: TChartModeProps
                         <Timeperiod newDesign portalNodeId={portalNodeId} onChange={onGranularity} />
                     </div>
                 </div>
+                {showTicksOnly ? (
+                    <div className={classNames('sc-chart-mode__section__text', { 'sc-chart-mode__section__text--mobile': isMobile })}>
+                        {t.translate('Only selected charts and time intervals are available for this trade type.')}
+                    </div>
+                ) : (
+                    <React.Fragment />
```

Comment:
Why do you need this line here?

Reply:
Just using ternary operation for rendering instead of short circuit evaluation to avoid render bug. Also, using <React.Fragment /> instead of null for the same reason.

Reply:
Reverted to using && instead